app/views/partials/header.ejs

```diff
@@ -5,6 +5,14 @@
     <meta charset="utf-8">
     <meta data-build-hash="<%=commitHash%>">
     <meta name="viewport" content="width=device-width">
+    <% if(typeof article !== 'undefined') { %>
+        <meta name="description" content="<%=article.description%>"">
+    <% } else if(typeof user !== 'undefined') { %>
+        <meta name="description" content="<%=user.bio%>">
```

Comment:
Google advices that meta tag shouldn't be a duplicate of already existing content on the page.

Reply:
I'm unaware of what else we could use for the user description, do you have any ideas?

Reply:
`<Username>'s profile page on <page name>, a blog about all things related to IT, created by passionates.`?